README.md

```diff
@@ -122,5 +121,5 @@ By default, the script will ask you to approve each override: `y` to override, `
 If you wish to approve all of the overrides at once, you can pass the `--bulk` option:
```

Comment:
Not sure from context if this example should or shouldn't have the --bulk option?

Reply:
Oops, good spot! Fixed.